README.md

```diff
@@ -7,37 +7,37 @@ Benchmarks for BigCache project
 go version
 go version go1.13 linux/amd64
 
-go run caches_gc_overhead_comparison.go
-
 Number of entries:  20000000
-GC pause for bigcache:  1.506077ms
-GC pause for freecache:  5.594416ms
-GC pause for map:  9.347015ms
+Number of repeats:  50
+GC pause for startup:  20.811µs
+GC pause for warmup:  3.715446ms
+GC pause for freecache:  38.607127ms
+GC pause for bigcache:  159.066277ms
+GC pause for map:  54.543926ms
```

Comment:
For smaller sets bigcaceh prefroms better
```
Number of entries:  1000000
Number of repeats:  50
GC pause for startup:  95.681µs
GC pause for warmup:  3.019482ms
GC pause for freecache:  70.401345ms
GC pause for bigcache:  37.559543ms
GC pause for map:  20.59779ms
```
we can try increasing number of shards

Reply:
This is one of the few instances where we could do some linear regression testing to validate what an optimised configuration should be. `y = mx + b` (heh) will be helpful for understanding what the optimised configurations should look like based off count and size.

I wonder how viable it would be have dynamic shard allocation, and if so, what would that look like?

Reply:
Yeah, that will be cool. Right now, I'm preparing a talk about bigcache history. Making it dependency less makes it easier to run benchmarks in previous go version (starting with Go 1.4) and comparing results.
Right now I think shards are definitely the point where we can get more performance.